File: v03/zad2/Pravougaonik.cpp
#include "Pravougaonik.hpp"

// Podrazumevani konstruktor
Pravougaonik::Pravougaonik(double a, double b) {
    this->a = a;
    this->b = b;
}

// Vraća prvu stranicu
double Pravougaonik::getA() const {
    return a;
}

// Vraća drugu stranicu
double Pravougaonik::getB() const {
    return b;
}

// Vraća obim pravougaonika
double Pravougaonik::getO() const {
    return 2 * (a + b);
}

// Vraća površinu pravougaonika
double Pravougaonik::getP() const {
    return a * b;
}

File: v09/zad1/Pravougaonik.java
package zad1;

public class Pravougaonik {
	
	// Polja
	private double a;
	private double b;
	
	// Konstruktor sa parametrima (izgenerisan)
	public Pravougaonik(double a, double b) {
		super();
		this.a = a;
		this.b = b;
	}
	
	// Konstruktor kopije
	public Pravougaonik(Pravougaonik p) {
		this.a = p.a;
		this.b = p.b;
	}
	
	// Geteri i seteri (izgenerisani)
	public double getA() {
		return a;
	}

	public void setA(double a) {
		this.a = a;
	}

	public double getB() {
		return b;
	}

	public void setB(double b) {
		this.b = b;
	}
	
	// Metoda koja vraća obim
	public double obim() {
		return 2 * a + 2 * b;
	}
	
	// Metoda koja vraća površinu
	public double povrsina() {
		return a * b;
	}
	
}
